lib/chess_piece_move.rb

```diff
@@ -12,6 +14,6 @@ def possible_positions
   private
 
   def possible_coordinates(coordinate)
-    figure_move_class.new(coordinate).possible_coordinates
+    figure_move_class.new(coordinate, ARGV[2]).possible_moves
```

Comment:
your class ChessPieceMove is now coupled with the way you interact with the app (by the console) - do you think it is possible to reduce coupling in that place